Library/src/main/java/com/pusher/android/notifications/PushNotificationRegistration.java

```diff
@@ -59,14 +59,20 @@ public PushNotificationRegistration(
         this.manifestValidator = manifestValidator;
     }
 
+    /**
+     * @deprecated this feature is being decommissioned, please use Pusher Beams instead
+     */
+    @Deprecated
     public void registerGCM(Context context, String defaultSenderId) throws ManifestValidator.InvalidManifestException {
         registerGCM(context, defaultSenderId, null);
     }
 
-    /*
-    Starts a PusherRegistrationIntentService, which handles token receipts and updates from
-    GCM
+    /**
+     * Starts a PusherRegistrationIntentService, which handles token receipts and updates from GCM
+     *
+     * @deprecated this feature is being decommissioned, please use Pusher Beams instead
```

Comment:
The word "this feature" can be misunderstood. Can we be more specific?

Reply:
What do you think of:
> Push Notifications no longer supported on this SDK. Please use Pusher Beams: www.pusher.com/beams

Reply:
👍 
("are no longer")

Reply:
Push Notifications are no longer supported *in* this SDK. Please use Pusher Beams, our new Push Notifications product: www.pusher.com/beams

is my suggestion. Mainly the use of the word with though.